src/node.cc

```diff
@@ -3786,6 +3790,16 @@ static void ParseArgs(int* argc,
     index += args_consumed;
   }
 
+#if HAVE_OPENSSL
```

Comment:
I think the code already does that, since the option is only recognised and parsed if `HAVE_OPENSSL` is defined.